### PR TITLE
Added code to display abandoned:name or razed:name when available for…

### PR DESCRIPTION
**Pull request by JonathanBittner**
… abandoned/razed railroads. This will override the mdoern highway or rail trail name
Fixes OpenRailwayMap/OpenRailwayMap-CartoCSS#44

### CODE REVIEW

**Comment by Nakaner** [comment]
Thank you for your contribution. Looks good at the first glance. I will have to test this before I push the "approve" button.

**Comment by JonathanBittner** [comment]
Some examples/test cases for you:
Name "Columbia Trail" abandoned:name "High Bridge Branch" https://openrailwaymap.org/?style=standard&lat=40.75231246487251&lon=-74.8188042640686&zoom=15
Name "North Shore Rail Trail" abandoned:name "Wading River Branch" https://openrailwaymap.org/?style=standard&lat=40.75231246487251&lon=-74.8188042640686&zoom=15

**Review by Nakaner — APPROVED** [review]

**Comment by JonathanBittner** [comment]
Thank you for approving my change. Is there anything I need to do to close this out. I'm still a GitHub newbie.

**Comment by Nakaner** [comment]
@JonathanBittner You don't have to do anything now. Otherwise I would have choosen "request changes". Another maintainer will merge this pull request.

File: standard.mml
Comment (DerDakon) [inline standard.mml:652]: please remove the whitespace at the end of the line

**Review by DerDakon — CHANGES_REQUESTED** [review]
Please remove the 2 bogus spaces and split the commit message: add 2 newlines after the first sentence so the first sentence will become the "title" of the commit and the second sentence is just additional text. Please also fix the typo ("mdoern") in the second sentence.

**Review by DerDakon — APPROVED** [review]

**Pull request merged**
